package.json

```diff
@@ -7,7 +7,7 @@
     "dev": "node server.js",
     "build": "next build",
     "start": "NODE_ENV=production node server.js",
-    "test": "NODE_ENV=test jest",
+    "test": "NODE_ENV=test jest --verbose --silent",
```

Comment:
I would suggest removing `--silent` in a development setting in order to allow useful console warnings and errors to display.

Reply:
Yeah, I added the tag for my own sanity. There are lots of type issues in the library that resulted into a lot of warnings, most which need to be addressed. I will remove that.

Reply:
I get what you are saying about sanity! 😉  But if there are issues in the library then we should expect those to be being picked up in tests there first so maybe we need to look into that.

Reply:
On a second look, I think the warnings were coming from Header component (expecting `i18nLng` to be an array), the matcher - `containsMatchingElement()` (which can easily be replaced by another matcher), and AppJs which is complaining about the component child supplied. Between @andreferreiradlw and I, we can resolve the `i18nLng` issue, the  `containsMatchingElement()` will be replaced but to be honest I dont have a clue about the AppJs warnings, at least not yet.